packages/tabs/package.json

```diff
@@ -31,6 +31,7 @@
   },
   "devDependencies": {
     "@kaizen/button": "^3.0.6",
+    "@kaizen/component-base": "^1.1.7",
```

Comment:
I think given the way this repo is distributed, it actually needs to be in `dependencies` 😅

Reply:
Oops, that's amended now.